src/tactic/ui/panel/base_table_layout_wdg.py

```diff
@@ -1566,8 +1575,7 @@ def get_button_row_wdg(my):
 
 
             # collection
-            #if SearchType.column_exists(my.search_type, "is_collection"):
-            if True:
+            if SearchType.column_exists(my.search_type, "is_collection"):
```

Comment:
You started with the convention _is_collection in the asset table.. I suggest sticking with that convention